packages/honduras/README.md

```diff
@@ -0,0 +1,39 @@
+# 🇭🇳 Honduras
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+Contains all the states of Honduras:
```

Comment:
It seems that they are called "departments" (source: https://en.wikipedia.org/wiki/Departments_of_Honduras)